.github/workflows/pr.yaml

```
@@ -23,7 +23,14 @@ jobs:
+      - name: Analyze
+        run: flutter analyze
 
       - name: Run tests
         run: |
-          flutter test --no-pub --coverage --test-randomize-ordering-seed=random
\ No newline at end of file
+          flutter test --no-pub --coverage --test-randomize-ordering-seed=random
+      
+      - name: Run warmup test set
+        run: |
+          flutter test --no-pub --dart-define=DK_ENABLE_ATTRIBUTE_WARM_UP=true test/warmup_test.dart
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)